### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rock3r**
Remove now-redundant TOC, fix a few typos, clarify a few items

### CODE REVIEW

**Review by mpgirro — CHANGES_REQUESTED** [review]
Of course GitHub releases a ToC feature just a day after one has been added to our Readme 😅

**Comment by rock3r** [comment]
What would you change? You didn't leave notes on that 😅

**Comment by mpgirro** [comment]
Ups, hand on...

File: README.md [hunk 1]
```diff
@@ -93,7 +75,7 @@ val podcast = PodcastRssParser.parse(rssFeedFile)
 ```
 
 The `parse()` function will return a parsed `Podcast?`, which may be `null` if parsing was unsuccessful. It will also throw an exception if parsing
-fails.
+fails catastrhopically (e.g., the source is a `File` that doesn't exist).
```
Comment (mpgirro) [inline README.md:78]: Typo :)

**Pull request merged**